src/main/java/org/opensearch/knn/index/codec/util/KNNCodecUtil.java

```diff
@@ -1,17 +1,22 @@
 /*
- * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
```

Comment:
we don't need to expand license description

Reply:
Sure, let me revert to a shorter header, seems this one has been added by signoff command automatically